tests/bluetooth/tester/src/btp_mesh.c

```diff
@@ -2066,6 +2082,161 @@ static uint8_t models_metadata_get(const void *cmd, uint16_t cmd_len,
 }
 #endif
 
+#if defined(CONFIG_BT_MESH_BRG_CFG_CLI)
+static uint8_t subnet_bridge_get(const void *cmd, uint16_t cmd_len,
+				 void *rsp, uint16_t *rsp_len)
+{
+	const struct btp_mesh_subnet_bridge_get_cmd *cp = cmd;
+	enum bt_mesh_subnet_bridge_state state;
+	int err;
+
+	err = bt_mesh_brg_cfg_cli_subnet_bridge_get(net.net_idx, sys_le16_to_cpu(cp->addr),
+						    &state);
+	if (err) {
+		LOG_ERR("err=%d", err);
+		return BTP_STATUS_FAILED;
+	}
+
+	LOG_DBG("Subnet Bridge state: %u", state);
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t subnet_bridge_set(const void *cmd, uint16_t cmd_len,
+				 void *rsp, uint16_t *rsp_len)
+{
+	const struct btp_mesh_subnet_bridge_set_cmd *cp = cmd;
+	enum bt_mesh_subnet_bridge_state state;
+	int err;
+
+	state = cp->val;
+
+	err = bt_mesh_brg_cfg_cli_subnet_bridge_set(net.net_idx, sys_le16_to_cpu(cp->addr),
+						    state, &state);
+	if (err) {
+		LOG_ERR("err=%d", err);
+		return BTP_STATUS_FAILED;
+	}
+
+	LOG_DBG("Subnet Bridge state: %u", state);
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t bridging_table_add(const void *cmd, uint16_t cmd_len,
+				  void *rsp, uint16_t *rsp_len)
+{
+	const struct btp_mesh_bridging_table_add_cmd *cp = cmd;
+	struct bt_mesh_bridging_table_entry entry;
+	struct bt_mesh_bridging_table_status rp;
+	int err;
+
+	LOG_DBG("");
+
+	entry.directions = cp->directions;
+	entry.net_idx1 = sys_le16_to_cpu(cp->net_idx1);
```

Comment:
Might this cause the unaligned address access exception? 
`cp->net_idx1` might have odd address, and there is an explicit cast to `uint16_t` in `sys_le16_to_cpu` that causes losing of the `packed` attribute from operated type.
@PavelVPV, @Balaklaka any opinion to such approach?

Reply:
Yeah, I think it may happen if `CONFIG_TRAP_UNALIGNED` is enabled (https://github.com/zephyrproject-rtos/zephyr/blob/386263cc668b10a2546b93a2a3e7444d75b7b05d/arch/arm/core/cortex_m/Kconfig#L389-L394).
In any case, I think it is better to align this fields in the packed struct.

Reply:
Should I then remove the "sys_le16_to_cpu", and just use "cp->netidx1" instead?

Reply:
Probably move `directions` field to the end of the structure or add an empty `uint8_t` field right after it.

Reply:
I'd suggest just to use proxy local variables like this:
```
entry.directions = cp->directions;
entry.net_idx1 = cp->net_idx1;
entry.net_idx1 = sys_le16_to_cpu(entry.net_idx1);
```
It will help to avoid any unaligned access problems.

Reply:
Why is a problem just here? I am trying to look through the code and there is nowhere that it is done like this. Should this not be a problem in provision_node too? The use of a proxy local variable look strange, and the question comes if we should do this other places too.
Remember this is also inside the tester code, and not production code. I have hard time seeing the value of doing this, and that it will raise more questions than doing good. And should be change this other places this might be a problem?

Reply:
The problem with this is that this approach requires a compiler to create a bunch of code to avoid unaligned access which is not good. And if `CONFIG_TRAP_UNALIGNED` is enabled, it will cause usage fault.

But I agree, the entire mesh tester is written without consideration of the unaligned access, so lets not nitpick here. If it should be fixed, it should be fixed separately for the all messages.